src/boot/z_std_dma.c

```diff
@@ -112,8 +112,8 @@ s32 DmaMgr_DmaRomToRam(uintptr_t rom, void* ram, size_t size) {
         ioMsg.size = buffSize;
 
         if (gDmaMgrVerbose == 10) {
-            PRINTF("%10lld ノーマルＤＭＡ %08x %08x %08x (%d)\n", OS_CYCLES_TO_USEC(osGetTime()), ioMsg.dramAddr,
-                   ioMsg.devAddr, ioMsg.size, MQ_GET_COUNT(&gPiMgrCmdQueue));
+            PRINTF("%10lld " T("ノーマルＤＭＡ", "Normal DMA") " %08x %08x %08x (%d)\n", OS_CYCLES_TO_USEC(osGetTime()),
```

Comment:
mentioned it before, but I'm not a fan of splicing translations into the middle of the string data. It looks like this particular file is doing so, while most other files do it this way:

```suggestion
            PRINTF(T("%10lld ノーマルＤＭＡ %08x %08x %08x (%d)\n, "%10lld Normal DMA %08x %08x %08x (%d)\n") , OS_CYCLES_TO_USEC(osGetTime()),
```

Reply:
in this case most of the string is % directives, plus this "normal dma" part is copypasted in four-five more lines below